### PR TITLE
Add scenario to create webhooks

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/main/java/clickup/entities/features/webhooks/WebHook.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+ */
+
+package clickup.entities.features.webhooks;
+
+import clickup.entities.Health;
+import clickup.entities.features.IFeature;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
```
Comment (Raymundogs) [inline src/main/java/clickup/entities/features/webhooks/WebHook.java]: unnecessary new line.

File: src/main/java/clickup/requests/WebHooksRequest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package clickup.requests;
```
Comment (GustavoZHA) [inline src/main/java/clickup/requests/WebHooksRequest.java:11]: add copyright

Comment (Raymundogs) [inline src/main/java/clickup/requests/WebHooksRequest.java:11]: Add copyright

File: src/test/java/cucumber/hooks/WebhookHooks.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package cucumber.hooks;
```
Comment (Raymundogs) [inline src/test/java/cucumber/hooks/WebhookHooks.java:11]: Add copyright

File: src/test/java/cucumber/hooks/WebhookHooks.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    private WebHooksRequest webHooksRequest;
+    private ScenarioTrash scenarioTrash;
+
+    public WebhookHooks(WebHooksRequest webHooksRequest, ScenarioTrash scenarioTrash) {
+        this.webHooksRequest = webHooksRequest;
+        this.scenarioTrash = scenarioTrash;
+    }
+
+    @Before(value = "@CreateWebhook")
+    public void createSpace() throws JsonProcessingException {
```
Comment (GustavoZHA) [inline src/test/java/cucumber/hooks/WebhookHooks.java]: change name of createSpace to createWebhook

File: src/test/java/cucumber/hooks/WebhookHooks.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        scenarioTrash.setScenarioTrash("webhook_id", webHooksRequest.createWebhook());
+    }
+
+    @After(value = "@DeleteWebhook")
+    public void deleteSpace() {
```
Comment (GustavoZHA) [inline src/test/java/cucumber/hooks/WebhookHooks.java]: change name of deleteSpace to deleteWebhook

**Review by Raymundogs — APPROVED** [review]

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**